### DISCUSSION TITLE
Specific port routing

### DISCUSSION THREAD

**Discussion by sgp729** [xjasonlyu/tun2socks#423] [Q&A]
I'm trying to proxy not all traffic but only packets with certain ports, I mark the packets I need but have no idea how to route them to the correct gateway:
```
iptables -t mangle -A PREROUTING -p udp --dport 50000:65000 -j MARK --set-mark 0xB
```
The desired behavior is: route packets marked with 0xB through dev tun0
How can I achieve this?

**Comment by xjasonlyu** [comment]
Interesting, I've never done this before, but it's theoretically feasible... 🤔
